CONTRIBUTORS.md

```diff
@@ -1,3 +1,4 @@
 From oldest to newest contributor, we would like to thank:
 
 - [Austin Morton](https://github.com/apmorton)
+- [Annie Tong](https://github.com/bbqroastcheese)
```

Comment:
I think this has already been added.

I wonder if when you made this PR, you weren't up to date with the latest changes in the `main` branch of the primary repository. Here are some things to check:

1. You've hit the "Sync fork" button in the page for your forked copy to ensure your fork has all the latest changes from the primary repository
2. On your local laptop, you've checked out the main branch using `git checkout main`, and then pulled the latest changes to your fork onto your laptop by running `git pull`.
3. For your `add_IDE_information` branch, you've merged the changes from the main branch by running `git merge main`. You can also avoid this in the future by doing steps 1 and 2 before creating a new branch.

You may get some merge conflicts in CONTRIBUTORS.md when you do this. Do your best to sort them out, but if you can't figure it out, post what you're seeing in Discord and someone will help you sort it out.